config/3.2/cp4waiops/templates/image-pull-secret.yaml

```diff
@@ -1,3 +1,4 @@
+{{- if or .Values.spec.eventManager.enabled .Values.spec.aiManager.enabled }}
```

Comment:
s/eventManager/eventManagerGateway

Ditto for others

Reply:
Done